config/routes.rb

```diff
@@ -1,8 +1,14 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   resources :genres
-  resources :users
-  
+
+  get '/users/:id', to:'users#show'
+  post '/users', to:'users#create'
+  put '/users/:id', to:'users#update'
+  delete'/users/:id', to:'users#destroy'
+  put 'users/:id/photo_path', to:'users#add_photo_path'
```

Comment:
I would call this method `update_photo_path` since you're actually updating it, not creating one